sendNewEHRtoVoters.js

```diff
@@ -0,0 +1,64 @@
+const rp = require('request-promise'),
+	utils = require('./etc/utils'),
+	tronUtils = require('./etc/tronUtils'),
+
+	db = global.healthportDb,
+	apiUrlForVotersList = `${process.env.TRON_SCAN_URL}api/vote`,
+	options = {
+	uri: '',
+	headers: {
+		'User-Agent': 'Request-Promise'
+	},
+	json: true // Automatically parses the JSON string in the response
+}
+
+if (process.env.NODE_ENV == 'production') {
+	sendNewEHRToVoters()
+}
+
+async function sendNewEHRToVoters() {
+	try {
+		//Getting voter data
+		options.uri = `${apiUrlForVotersList}?limit=50&candidate=${process.env.VOTER_ACCOUNT}`
+		const response = await rp(options)
+
+		for (let i = 0; i < response.data.length; i++) {
+			const oldTokenBalance = await tronUtils.getHealthportOldTokenBalance(response.data[i].voterAddress)
+			const newTokenBalance = await tronUtils.getTRC10TokenBalance('', response.data[i].voterAddress)
+			if (newTokenBalance == 0 && oldTokenBalance > 0 && (newTokenBalance != oldTokenBalance)) {
+				//Excluding potential hacker
+				if (response.data[i].voterAddress != 'TB5ZDGoqiL89HiNMWo26b6ciZUUv6QsfWQ') {
```

Comment:
Ignore case sensitivity for this address.

Reply:
toLowerCase added for comparison